fala/apps/adviser/tests/test_search_view_function.py

```diff
@@ -65,6 +65,35 @@ def test_category_search_has_no_previous_button(self):
         self.assertNotContains(response, '<div class="govuk-pagination__previous">')
 
 
+@override_settings(FEATURE_FLAG_NO_MAP=True)
+class PaginationTest(SimpleTestCase):
+    client = Client()
+    url = reverse("search")
+
+    data = {"postcode": "PE30", "categories": "deb", "page": "2"}
+
+    def test_prev_pagination_link_contains_category(self):
```

Comment:
we could, but I'm not sure how valuable it would be. I'm not sure how looking for the absence of something is useful. Maybe the test should be tighter in terms of what it is expecting so that its obvious?